src/main/kotlin/io/embrace/android/intellij/plugin/ui/components/EmbBlockCode.kt

```diff
@@ -1,11 +1,16 @@
 package io.embrace.android.intellij.plugin.ui.components
 
+import io.embrace.android.intellij.plugin.manager.EmbraceIntegrationDataProvider
+import io.embrace.android.intellij.plugin.network.HttpClient
 import java.awt.Color
 import java.awt.Font
 import javax.swing.JLabel
 
 internal class EmbBlockCode(block: CODE_BLOCK) : JLabel() {
 
+    private val httpClient = HttpClient()
```

Comment:
I think we can inject this, so we can use the same instance in all components, forms. Or maybe we can pass the Embrace version and do the call outside of the component.